index.bs

```diff
@@ -280,7 +281,6 @@ A <dfn>registry entry</dfn> is a record with the following fields:
 1. Let _instance_ be Instantiation(_loader_, _optionalInstance_, _source_).
 1. ReturnIfAbrupt(_instance_).
 1. // TODO: edge case: what if _instance_ is a thenable function?
-1. Fulfill _entry_.[[Instantiate]] with _instance_.
```

Comment:
`CommitInstantiated()` is called from:

* `RequestInstantiate()`, in which case the promise in `[[Instantiate]]` will be resolved implicitely, and there is not need to fulfilled manually.
* `FulfillInstantiate()`, in which case the promise should be fulfillled manually with the provided value.

This change seems to be correct.